RemoteHealthcare/ConsoleGUI.cs

```diff
@@ -1,111 +1,149 @@
+            int entryAmount = 0;
+            while (!validEntery)
+            {
+                Console.Clear();
+
+                Console.Write("Hoeveel data pakketen wil je ontvangen: ");
+                try
+                {
+                    entryAmount = int.Parse(Console.ReadLine());
+                    if (entryAmount! > 0)
+                    {
+                        throw new Exception();
+                    }
+
+                    validEntery = true;
+                }
+                catch (Exception)
+                {
+                    Console.BackgroundColor = ConsoleColor.DarkRed;
+                    Console.WriteLine("Getal invoer is niet correct probeer opnieuw (Druk op een knop)");
+                    Console.BackgroundColor = ConsoleColor.Black;
+                    Console.ReadKey();
+                }
+            }
+            Console.Clear();
+            program.hrManager.MakeConnection(entryAmount);
+        }
+
+        private void ReadBike()
+        {
+            Console.Clear();
+            Console.Write("Wat is het serie nummer van de fiets? : ");
+            string serie = Console.ReadLine();
+            bool enteryValid = false;
+            int amountEntry = 0;
+            while (!enteryValid)
+            {
+                Console.Clear();
+
+                Console.Write("Hoeveel data pakketen wil je ontvangen: ");
+                try
+                {
+                    amountEntry = int.Parse(Console.ReadLine());
+                    if (amountEntry! > 0)
+                    {
+                        throw new Exception();
+                    }
+
+                    enteryValid = true;
+                }
+                catch (Exception)
+                {
+                    Console.BackgroundColor = ConsoleColor.DarkRed;
+                    Console.WriteLine("Getal invoer is niet correct probeer opnieuw (Druk op een knop)");
+                    Console.BackgroundColor = ConsoleColor.Black;
+                    Console.ReadKey();
+                }
+            }
+
+            Console.Clear();
+            program.bikeManager.MakeConnectionAsync(serie, amountEntry).Wait();
+        }
+
+        private void RunStep()
+        {
+            Console.Clear();
+            PrintLine(ConsoleColor.DarkRed, "Hieronder 1 item gegenereerd door de simulator.");
+            this.program.bikeManager.SimRunStep();
+            PrintLine(ConsoleColor.DarkRed, "Druk op een knop om door te gaan.");
+            Console.ReadKey();
+        }
+
+        private void StartSim()
+        {
+            Console.Clear();
+            this.program.bikeManager.StartSim();
+        }
+
+        private static void PrintLine(ConsoleColor color, string toPrint)
+        {
+            Console.BackgroundColor = color;
+            Console.WriteLine();
+            Console.BackgroundColor = ConsoleColor.Black;
+        }
 
-        public string mainMenu()
+        public char MainMenu()
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)